src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -511,6 +511,14 @@ Result Device::CreateApiObjects(const grfx::DeviceCreateInfo* pCreateInfo)
         extensionStructs.push_back(reinterpret_cast<VkBaseOutStructure*>(&descriptorIndexingFeatures));
     }
 
+    // VK_EXT_scalar_block_layout
```

Comment:
I think it might be - splitting the PRs wasn't an exact science, so I aimed to do it file by file. I'm going to update the PR description / commit comment to explain this change.